advanced_functionality/huggingface_byo_scripts_and_data/huggingface-custom-text-summarizer.ipynb

```diff
@@ -0,0 +1,2886 @@
+{
```

Comment:
<p>To make cleanup more thorough, add in <code>predictor.delete_model()</code> as well. Also, make sure delete_model() is executed first before delete_endpoint() to avoid any errors </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>